conf/app.routes

```diff
@@ -2,7 +2,8 @@
 
 POST        /traders/:eori/movements/:movementType/:movementId/messages/:messageId             uk.gov.hmrc.transitmovementsrouter.controllers.MessagesController.outgoing(eori: EoriNumber, movementType: MovementType, movementId: MovementId, messageId: MessageId)
 
-POST        /movements/:movementId/messages/:messageId                                                uk.gov.hmrc.transitmovementsrouter.controllers.MessagesController.incomingLargeMessage(movementId: MovementId, messageId: MessageId)
+POST        /movements/:movementId/messages/:messageId                                         uk.gov.hmrc.transitmovementsrouter.controllers.MessagesController.incomingLargeMessage(movementId: MovementId, messageId: MessageId)
 
 POST        /movements/:ids/messages                                                           uk.gov.hmrc.transitmovementsrouter.controllers.MessagesController.incoming(ids: ConversationId)
 
+POST        /movements/sdes/callback                                                           uk.gov.hmrc.transitmovementsrouter.controllers.MessagesController.handleSdesResponse()
```

Comment:
@drnaylor-oc  I think this should be PATCH call as we need to change only status with db.  What do you think?

Reply:
SDES will only send a POST.